website/docusaurus.config.js

```diff
@@ -23,9 +23,14 @@ module.exports = {
       logo: {
         alt: 'Gatekeeper logo',
         src: 'img/logo.svg',
-        href: 'https://open-policy-agent.github.io/gatekeeper/website/docs/',
+        href: 'https://open-policy-agent.github.io/gatekeeper/website/',
       },
       items: [
+        {
+          to: 'docs/',
+          label: 'Docs',
```

Comment:
```suggestion
          label: 'Documentation',
```
OPA's website uses `Docs` but Kubernetes has "Documentation", which one do folks prefer?

Reply:
No preference on my side, willing to defer to what others think

Reply:
`Docs` +1